frontend/modules/viewpoints/viewpoints.sagas.ts

```diff
@@ -214,17 +214,16 @@ export function* showViewpoint({teamspace, modelId, view, ignoreCamera}) {
 			yield Viewer.isViewerReady();
 
 			const viewpoint = view.viewpoint;
-			if (!viewpoint) {
-				return;
-			}
 
 			if (viewpoint?.up && !ignoreCamera) {
 				yield put(ViewerGuiActions.setCamera(viewpoint));
 			}
 
 			const clippingPlanes = view.clippingPlanes || get(view, 'viewpoint.clippingPlanes');
 
-			yield put(TreeActions.setHiddenGeometryVisible(viewpoint.hideIfc !== false));
```

Comment:
I think we need the original logic.

If the viewpoint didn't specify hideIfc (which is possible for viewpoints created by user via API I believe), it should be true by default. The modified version wouldn't capture this.